extension/build.gradle

```diff
@@ -235,8 +235,19 @@ task copyBuildServerJars(type: Copy) {
   mustRunAfter copyDocs
 }
 
+// TODO: This task can be removed once the build server is published.
+task copyBuildServerJarsToGradleServer(type: Copy) {
+  dependsOn ':extension:buildBuildServer'
+  from('./build-server-for-gradle/server/build/libs/') {
+    include '**/*.jar'
+    include '**/init.gradle'
+  }
+  into '../gradle-server/build/libs/'
```

Comment:
If directly copying those jars into `gradle-server/build/libs/`, Then the exclude makes no sense